.gitignore

```diff
@@ -13,6 +13,7 @@ buildNumber.properties
 
 .gradle/
 build/
+out/
```

Comment:
Shouldn't everything be in `build/`? (I'm not so familiar with Gradle so it's a real question)

Reply:
Totally right. out/ was an IDE setting problem. I'm not so familiar with Gradle too tho.